hassio-google-drive-backup/backup/static/layouts/partials/modals/snapshot/new.jinja2

```diff
@@ -1,32 +1,37 @@
 <div id="snapshotmodal" class="modal">
```

Comment:
> To support the changes you've made, I'm going to need to also update a laundry list of things. I don't think any of this should block this submission but they're things I'll need to do before merging this into main and making a release. Again I'm not asking you to do any of this, its more of a list of "to-do's" for me before making a release of these changes. I'll dedicate the dev branch to implementing these for the time being, and make bug fixes elsewhere.

I could probably do some of it!
Like:
- Snapshot names that get truncated will need to have the full name displayed in a tooltip (or somewhere else) if they get truncted in the new snapshot card.
- The "?" info icon on the snapshot dialog can be removed now that the whole thing opens the detail dialog.
- Drop shadows aren't very visible using a dark theme, which makes modals hard to use and cards difficult to see. I think this too is a pre-existing problem.

About the drop shadows, they works ok when you use a background color slightly paler than black. But on a pitch black background, you are right; they are just not visible. I didn't want to use a white drop shadow on darker backgrounds, as it made the UI less appealing. But I could probably make something in-between for that case.

Reply:
Oh yes I'd absolutely appreciate the help.  I just don't think any of that work should block this PR (its big plenty big already).

I'm also not sure what exactly to do about the drop shadows, since I agree the pure white ones look *awful* on pure black. I was meaning to fiddle around with the colors to see if I can get something to look nice.

Reply:
- [x] Snapshot names that get truncated will need to have the full name displayed in a tooltip (or somewhere else) if they get truncted in the new snapshot card.
- [x] The "?" info icon on the snapshot dialog can be removed now that the whole thing opens the detail dialog.
- [x] Drop shadows aren't very visible using a dark theme, which makes modals hard to use and cards difficult to see. I think this too is a pre-existing problem.

---

So for the drop shadows, what I did is that I use white shadows only on pitch-black background. The rest of the time the shadows are black:

### Absolute black:

![image](https://user-images.githubusercontent.com/9013072/110260680-37466f80-7f7b-11eb-8108-94a94c0f984d.png)

![image](https://user-images.githubusercontent.com/9013072/110260692-41686e00-7f7b-11eb-984a-26932c81cbe7.png)

### Lighter black:

![image](https://user-images.githubusercontent.com/9013072/110260719-5b09b580-7f7b-11eb-85a4-0e2fd885f38c.png)

![image](https://user-images.githubusercontent.com/9013072/110260724-62c95a00-7f7b-11eb-97cd-94e6d2040b1d.png)

### White:

![image](https://user-images.githubusercontent.com/9013072/110260740-6b219500-7f7b-11eb-9f13-3bc44362e25e.png)

![image](https://user-images.githubusercontent.com/9013072/110260743-6fe64900-7f7b-11eb-8eb3-2e88bc8331d0.png)